webpack.config.js

```diff
@@ -45,7 +45,7 @@ const config = {
 							loader: 'sass-loader',
 							query: {
 								includePaths: [ 'editor/assets/stylesheets' ],
-								data: '@import "variables";',
+								data: '@import "variables"; @import "_mixins";',
```

Comment:
Two things:

- IIRC, SASS doesn't care whether the file is prefixed with an `_` and will allow `@import "mixins";` here?
- We should choose one or the other for the file name convention and apply it to `variables`